sdk/cdsclient/interface.go

```diff
@@ -14,6 +14,17 @@ type Filter struct {
 	Name, Value string
 }
 
+// TemplateClient exposes templates functions
+type TemplateClient interface {
+	TemplateGet(groupName, templateSlug string) (*sdk.WorkflowTemplate, error)
```

Comment:
try to return a concrete type instead of a pointer

Reply:
I want this to return nil if not found but not Zero value for WorkflowTemplate.